regulations.tex

```diff
@@ -3,11 +3,58 @@ \section{Regulations}
 
 \begin{enumerate}
 \item The Judge's decision is final.
+\item All robots must be safe.
+  \begin{enumerate}
+    \item This is defined considering safety concerns including, but not limited
+          to:
+      \begin{enumerate}
+        \item sharp edges;
+        \item the effects of impact at speed;
+        \item fire risks from the battery (see Regulation~\ref{reg:lipo}).
+      \end{enumerate}
+    \item No robots will be permitted to compete without passing a safety and
+          compliance inspection.
+    \item \staff may reinspect your robot and invalidate previous inspections at
+          any time.
+  \end{enumerate}
 \item Any assistance from \staff is provided without guarantees.
 \item Competitors are expected to behave within the spirit of good
       sporting conduct.
+\item Competitors must take reasonable measures to avoid their robot damaging
+      the arena, or anything within it, including other robots. This is a
+      non-contact sport.
+\item Competitors are not permitted in the arena during the competition,
+      except to lean in to start robots or where directed by match officials.
```

Comment:
Are we intending for competitors or volunteers to start robots.

Reply:
I defer this to @prophile